pom.xml

```diff
@@ -2526,7 +2526,7 @@
         <carbon.registry.version>4.7.24</carbon.registry.version>
         <carbon.registry.imp.pkg.version>[4.7.0, 5.0.0)</carbon.registry.imp.pkg.version>
         <!-- Analytic Commons version-->
-        <carbon.analytics.common.version>5.2.18</carbon.analytics.common.version>
+        <carbon.analytics.common.version>5.3.7</carbon.analytics.common.version>
```

Comment:
Bump the synapse version to the latest.

Reply:
Upgraded the Synapse version to the latest with https://github.com/wso2/carbon-mediation/pull/1695/commits/340fce85a4e30c165feb701ca8d3253ca6fadd05 and kept the `carbon.analytics.common.version` at 5.3.7 since it is the latest released version in 5.3.x.

Thank you.